docs/core/compatibility/toc.yml

```diff
@@ -20,6 +20,8 @@ items:
         href: aspnet-core/8.0/addratelimiter-requirement.md
     - name: Containers
       items:
+      - name: Containers default to use the 'latest' tag
+        href: containers/8.0/default-image-tag.md
```

Comment:
This is worth discussing with @richlander - mt opinion is that I don't think this change should go into the `Containers` space because all of those changes are about the container base images that our customers use for their own applications. This breaking change notice is about .NET SDK-shipped tooling that applies irrespective of the base image that's used, so I'd push for being part of the `SDK` topic.

Reply:
It's unclear how customers think about these things and some of the breaks that affect the SDK scenario are from the base images. It seems like seeing those all in one place is probably best, but I'm good either way.

Reply:
How about this:

* changes that we 'inherit' from the dotnet-docker space (like APP_UID, ASP.NET Ports, etc) we can document in `Containers`
* tooling/integration/etc breaking changes go to `SDK`